src/DataFixtures/BaseFixture.php

```diff
@@ -4,13 +4,30 @@
 
 namespace Bolt\DataFixtures;
 
+use Bolt\Configuration\Areas;
+use Bolt\Configuration\Config;
 use Doctrine\Bundle\FixturesBundle\Fixture;
+use Symfony\Component\Finder\Finder;
+use Tightenco\Collect\Support\Collection;
+use Webmozart\PathUtil\Path;
 
 abstract class BaseFixture extends Fixture
 {
     private $referencesIndex = [];
     private $taxonomyIndex = [];
 
+    /** @var Config */
+    protected $config;
+
+    /** @var Areas */
+    protected $areas;
+
+    public function __construct(Config $config, Areas $areas)
+    {
+        $this->config = $config;
+        $this->areas = $areas;
```

Comment:
Why this constructor is added here when none of these dependencies is used in this class? 

Areas should not be a part of BaseFixture. move it to trait or to another abstract class